### ISSUE TITLE
Speed and memory benchmarks for complete overfitting

### ISSUE THREAD

**Issue by ogencoglu** [neurodata/treeple#341]
Are there speed and memory benchmarks (e.g. compared to `scikit-learn`, `obliquetree` etc.) for overfitting a dataset completely (no limit on tree depth)? This is usually a fair comparison for speed and memory without having to care about the accuracy part.

**Comment by adam2392** [comment]
Hi @ogencoglu thanks for the question. 

No not yet, and there are definitely speed limitations for certain tree implementations in `treeple` when it compares to scikit-learn for a variety of technical reasons. We would love to run it tho!

What is `obliquetree`? I could not find that package online?

**Comment by ogencoglu** [comment]
Here:
- https://obliquetree.readthedocs.io/en/latest/
- https://github.com/sametcopur/obliquetree

**Comment by adam2392** [comment]
Oh interesting I have not seen that yet. Our oblique tree implementation is slightly different tho just glancing at the implementation and description. 

Would indeed be great to benchmark across the community!
